.github/workflows/build_linux_packages.yml

```diff
@@ -83,10 +90,12 @@ jobs:
           cmake --build build --target therock-archives therock-dist
 
       - name: Test Packaging
+        continue-on-error: ${{ inputs.expect_failure }}
         run: |
           ctest --test-dir build --output-on-failure
 
       - name: Report
+        continue-on-error: ${{ inputs.expect_failure }}
         if: ${{ !cancelled() }}
```

Comment:
This is quite a bit of plumbing for the continue-on-error, which may let unexpected failures through too. Should we move these builds that don't succeed yet to scheduled jobs instead of running on `push` or `pull_request` events instead?